talaiot/src/test/kotlin/com/cdsap/talaiot/metrics/MetricsConfigurationTest.kt

```diff
@@ -14,6 +14,8 @@ class MetricsConfigurationTest : BehaviorSpec({
             assert(metrics.count { it is RootProjectNameMetric } == 1)
             assert(metrics.count { it is GradleRequestedTasksMetric } == 1)
             assert(metrics.count { it is GradleVersionMetric } == 1)
+            assert(metrics.count { it is HostnameMetric } == 1)
+            assert(metrics.count { it is OsManufacturerMetric } == 1)
         }
```

Comment:
Suggestion: these are being tested as if they belong to the `default()` function, but they are actually part of `environment()` function.

Maybe we could separate them in their own group:

```suggestion
        }
        
        `when`("environment metrics are configured") {
            val metricsConfiguration = MetricsConfiguration()
            val metrics = metricsConfiguration.performance().build()
            
            assert(metrics.count { it is HostnameMetric } == 1)
            assert(metrics.count { it is OsManufacturerMetric } == 1)
            assert(metrics.count { it is PublicIpMetric } == 1)
            assert(metrics.count { it is DefaultCharsetMetric } == 1)
        }
```

I've also added `PublicIpMetric` and `DefaultCharsetMetric` in the checks, since they are part of `environment()`.